src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -86,7 +100,7 @@ public void configurePipeline(PipelineConfigurer configurer) throws IllegalArgum
 
     Schema iSchema = configurer.getStageConfigurer().getInputSchema();
 
-    if (!config.field.equalsIgnoreCase("all")) {
+    if (!config.field.equalsIgnoreCase("*")) {
```

Comment:
UI was having trouble in sending `*` in tracker search api. We should confirm that this will work from their end.